tests/bats/audit/audit-devices.bats

```diff
@@ -8,6 +8,7 @@ load ../helpers
     #
     export GRAPHQL_QUERY_FILE=/tests/fixtures/audit/enable_audit_device.graphql
     run vault-manager
+    echo $output
```

Comment:
is this change for all test files or just these two? keep it consistent if you are adding this as a test debug feature